.github/workflows/scheduled.yaml

```
@@ -38,6 +38,8 @@ jobs:
       strategy: ${{ matrix.test-strategy }}
       additional-env-vars: |
         PKG_SYSREQS_DRY_RUN=true
+      extra-deps: |
+        MultiAssayExperiment (>= 1.32.0);SummarizedExperiment (>= 1.36.0);teal.slice (>= 0.5.1.9021)
```

Following similar PR, can't it be replaced by:

```suggestion
      extra-deps: |
        matrixStats (>= 1.5.0);teal.slice (>= 0.5.1.9021)
```